apps/wpcom-block-editor/src/common/disable-nux-tour.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * External dependencies
+ */
+import { dispatch } from '@wordpress/data';
+import '@wordpress/nux'; //ensure nux store loads
+
+dispatch( 'core/nux' ).disableTips();
```

Comment:
Do we want to disable all tips (as we're doing here), or just mark the "wonderful world of blocks" tip as dismissed?

Reply:
If we do want the former, should we also remove the ellipsis menu > Options > Enable Tooltips option, since it's odd to be able to go in and enable it, then have the next page load remove it again with no explanation.

<img width="830" alt="Screen Shot 2019-05-23 at 12 32 23 PM" src="https://user-images.githubusercontent.com/349751/58281023-e722fc80-7d56-11e9-8b11-4ec347acafff.png">

Reply:
I think it's okay to leave this for now. Clicking on Enable Tips will re-display tips on the current load. If it's confusing we can remove it in a follow up.

Reply:
Yeah I left it re-enable-able on purpose... in case someone wants to go through the tour manually, they can do that. It won't persist but I think that's okay.